tiled/server/jwt_auth.py

```diff
@@ -0,0 +1,123 @@
+from datetime import datetime, timedelta
+from typing import Optional
+
+from fastapi import Depends, APIRouter, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from jose import JWTError, jwt
+
+from passlib.context import CryptContext
+
+from pydantic import BaseModel
+
+# to get a string like this run:
+# openssl rand -hex 32
+SECRET_KEY = "09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7"
```

Comment:
Marking this to be sure we revisit it before merging. This should come from Settings and ultimately from the environment. It should also be a `;`-separated list of keys to support key rotation. The first entry will be used for encoding but any of them will be valid for deciding.

Reply:
Can we make this into an issue and do this later?